packages/components/src/components/buttonlink/buttonlink.component.ts

```diff
@@ -0,0 +1,177 @@
+import { CSSResult, html, PropertyValues } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import styles from './buttonlink.styles';
+import Link from '../link';
+import {
+  BUTTON_COLORS,
+  BUTTON_TYPE_INTERNAL,
+  BUTTON_VARIANTS,
+  DEFAULTS,
+  ICON_BUTTON_SIZES,
+  PILL_BUTTON_SIZES,
+} from '../button/button.constants';
+import type { IconNames } from '../icon/icon.types';
+import type { ButtonColor, ButtonVariant, IconButtonSize, PillButtonSize } from '../button/button.types';
+import type { LinkSize } from '../link/link.types';
+import Button from '../button/button.component';
```

Comment:
Should we import the styles file directly from button instead of the component file?
```suggestion
import buttonStyles from '../button/button.styles';
```

Reply:
Yes, we can go with this approach, but we haven’t been following it consistently so far. If we decide to adopt it, we should apply the change across all relevant components to maintain consistency.